In [1]:
p = 0.5 ** 4
p

0.0625

The probability for all three scenarios is 6.25% because the likelihood of any result on any toss is 50%.

In [2]:
p_2 = 24/45
p_2

0.5333333333333333

The probability of not selecting a man (or, of selecting a woman) is 24/45, or 53.33%.

In [4]:
p_3 = 0.1 * .00005
p_3

5e-06

The probability of Bernice experiencing a plane crash this year is 0.0005%.

4. This methodology shows selection bias, because data is only being sampled from people who click on links and take the time to complete the survey.

Drill Set 2 Intuitions
1. 10.5%
2. 98%
3. 90%
4. 9.5%

In [5]:
p_pos = .98 * .005 + .1 * .995
p_pos

0.1044

2. 98%, given


3. There's a 10% chance an uninfected person will be given a positive result. The only other option is a negative result, meaning the answer is 90% (100% - 10%).

In [6]:
p_misclassify = .02 * .005 + .1 * .995
p_misclassify

0.09960000000000001

In [7]:
pwd

'C:\\Users\\danehooker'